kokoro/scripts/build/build_package.ps1

```diff
@@ -70,3 +70,5 @@ if ($env:KOKORO_ROOT_JOB_TYPE -eq 'CONTINUOUS_INTEGRATION') {
 # Copy the .goo file from $env:KOKORO_ARTIFACTS_DIR/out to $env:KOKORO_ARTIFACTS_DIR/result.
```

Comment:
comment is now kind of stale; could you add a comment about what these pdb and dll files are for or why we are saving them?